com.unity.netcode.gameobjects/CHANGELOG.md

```diff
@@ -34,6 +34,7 @@ Additional documentation and release notes are available at [Multiplayer Documen
 - Fixed `NetworkAnimator` issue where it was not taking the animation speed or state speed multiplier into consideration. (#1946)
 - Fixed `NetworkAnimator` issue where it was not properly synchronizing late joining clients if they joined while `Animator` was transitioning between states. (#1946)
 - Fixed `NetworkAnimator` issue where the server was not relaying changes to non-owner clients when a client was the owner. (#1946)
+- Fixed issue where `PacketLoss` would return the packet loss over a connection lifetime instead of a single frame. (#2004)
```

Comment:
Minor, but might want to precise what kind of packet loss this is (e.g. packet loss gathered by metrics for the tools package). Reading this, a user could realistically think we have some kind of official API to query packet loss.

Reply:
Will do.